packages/react-native-bridge/index.js

```diff
@@ -515,6 +515,20 @@ export function sendEventToHost( eventName, properties ) {
 	);
 }
 
+/**
+ * Shows Android's soft keyboard if there's a TextInput focused and
+ * the keyboard is hidden.
+ *
+ * @return {void}
+ */
+export function showAndroidSoftKeyboard() {
```

Comment:
I saw that this function is called in cases unrelated to dismissing a bottom sheet. For example, it's called when creating/selecting Paragraph blocks, as well as selecting the post title. Specifically, the call is made from `componentWillUnmount` function of three different bottom sheets:
- `link-settings-modal`
- `block-transformations-menu`
- `block-actions-menu`
- Suggestion picker (i.e. xpost, mentions).

I haven't spotted any undesired side effects due to this, but makes me wonder if we should trigger it when bottom sheets are unmounted. I'd expect that it's only called when it's actually dismissed by a user action. Could we rely only upon the `onCloseBottomSheet` event to trigger `showAndroidSoftKeyboard` when the bottom sheet is dismissed?

**Steps:**
1. Create a post.
2. Select the first paragraph.
3. Select the post title.
4. Observe that the function is called.

Reply:
Good catch, it appears some modals are being unmounted so I added [this check](https://github.com/WordPress/gutenberg/pull/57069/commits/79d62e82b58ad6355ce1576b05a9c131ac494145#diff-7c783d6dab99fab7810467451bf92877850e8c2f4808af9dd836ca828fdd9821R224-R229).

Reply:
Ah, interesting. Thanks for addressing it.